lib/internal/event_target.js

```diff
@@ -691,6 +698,29 @@ class EventTarget {
     }
   }
 
+  [kRemoveListenerHelper](type, listenerWrapper) {
+    const root = this[kEvents].get(type);
+    if (root === undefined || root.next === undefined)
+      return;
+
+    const capture = listenerWrapper.capture === true;
+
+    const listener = listenerWrapper.weak ? listenerWrapper.listener.deref() : listenerWrapper.listener;
```

Comment:
When would `listenerWrapper.weak` ever be falsey?

Reply:
renamed to indicate it's only for weak listener and updated the function